cmd/monaco/generate/dependencygraph/command.go

```diff
@@ -71,6 +72,8 @@ func Command(fs afero.Fs) (cmd *cobra.Command) {
 
 	cmd.Flags().StringVarP(&outputFolder, "output-folder", "o", "", "The folder generated dependency graph DOT files should be written to. If not set, files will be created in the current directory.")
 
+	cmd.Flags().BoolVar(&jsonIDs, "json-ids", false, "Set to generate a DOT file encoding each node's coordinate as JSON, instead of the default string representation. This can be useful when processing generated DOT files automatically.")
```

Comment:
I would go with a string value. For now we would have:
* string (default)
* json. 

I don't expect to have more options, but we keep the flexibility to easily add another one without breaking anything.

Reply:
Good idea! I've added a commit changing this, please re-review/approve